library/src/main/java/com/orm/SugarRecord.java

```diff
@@ -75,6 +75,14 @@
         return findById(type, Long.valueOf(id));
     }
 
+    public static <T> T first(Class<T>type){
+        return findById(type, 1);
```

Comment:
The first object in the database may not necessarily have an `id` of `1`. I think a better implementation would do a find with an order by ascending `id` and limit the query to a single item.